EDDI/EDDI.cs

```diff
@@ -1305,35 +1301,11 @@ private bool eventVehicleDestroyed(VehicleDestroyedEvent theEvent)
 
         private bool eventStatus(StatusEvent theEvent)
         {
-            if (Instance.CurrentStatus != theEvent.status)
+            if (theEvent.status.supercruise == true)
```

Comment:
comparison against true is unsafe, please use `!=false` or simply `if (theEvent.status.supercruise)`